pkg/windows/windows.go

```diff
@@ -245,8 +248,10 @@ func (vm *windows) Run(cmd string, psCmd bool) (string, error) {
 
 	out, err := vm.interact.run(cmd)
 	if err != nil {
-		// Hack to not print the error log for "sc.exe qc" returning 1060 for non existent services.
-		if !(strings.HasPrefix(cmd, serviceQueryCmd) && strings.HasSuffix(err.Error(), serviceNotFound)) {
+		// Hack to not print the error log for "sc.exe qc" returning 1060 for non existent services
+		// and hack to not print error when the command takes too long to return after deleting HNS networks.
```

Comment:
@mansikulkarni96 I am wondering if the issue you are running into is that when you delete the HNS networks, it causes the networks to be reconfigured thereby losing connectivity. This could be manifesting itself by taking too long to return. Please see [comment](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/windows/windows.go#L375).

Reply:
I understand, that is exactly why this hack is needed. Are you suggesting a workaround for that, not sure I got that part?

Reply:
Point I am making is that the hack is too generic.

Reply:
IOW wouldn't we want to know a command other than deleting the HNS networks took too long to run? By suppressing the error message across the board, you are hiding that information.